docker.local/build.sharder/b0docker-compose.yml

```diff
@@ -62,6 +62,7 @@ services:
       - ../config:/0chain/config
       - ../sharder${SHARDER}/log:/0chain/log
       - ../sharder${SHARDER}/data:/0chain/data
+      - ../sharder${SHARDER}/data/blocks:/path/to/root/directory
```

Comment:
We shall avoid using this dir path to store blocks, it is simply `/0chain/data/blocks`. We need to update the 0chain.yaml config to use the correct path. Or make it easier to deploy to store data into the `/0chain/data` the same way before we do the enhanced blocks saving. In that, we use the `storage` config only when we need to store blocks in different dir, or use cache. Otherwise, leave it blank to store blocks in `/0chain/data/blocks` by default.